static/js/presence.js

```diff
@@ -33,6 +33,9 @@ exports.is_not_offline = function (user_id) {
 };
 
 exports.get_status = function (user_id) {
+    if (user_id === page_params.user_id) {
```

Comment:
We should add a simple unit test for this.  Right now the `presence` stuff is tested in `frontend_tests/node_tests/activity.js` for legacy reasons, so you can add a test there for now.  Eventually we should split out a new test module, but that can be a separate project down the road.